requirements.txt

```diff
@@ -1,2 +1,8 @@
-requests
-pycairo
\ No newline at end of file
+certifi==2022.9.24
+charset-normalizer==2.1.1
+idna==3.4
+pycairo==1.21.0
+PyGObject==3.42.2
+PyGObject-stubs==1.0.0
+requests==2.28.1
+urllib3==1.26.12
```

Comment:
What are all these for?

Reply:
There dependencies packages are needed for me to run it on IDE while developing and debugging.